src/e2e/pages/authPage.ts

```diff
@@ -62,7 +60,7 @@ export class AuthPage {
       process.env.E2E_TEST_ACCOUNT_PASSWORD as string,
     );
     await this.ageInputField.type("31");
-    await this.continue();
+    await this.continue({ waitForURL: "**/accounts.stage.mozaws.net/**" });
```

Comment:
From my testing, this is the only place that actually needs a `waitForURL`, so maybe we could remove it from the `continue`? It's just the bit where we get redirected back to the Monitor that is hard for Playwright.

We can also avoid hardcoding the FxA stage URL here by waiting for an additional redirect, like this:

```js
await this.page.waitForURL(`${process.env.E2E_TEST_BASE_URL}/**`);
```

Reply:
That works! I put them there to make sure this is the source of failure while debugging in CI — updated in [eb173c9](https://github.com/mozilla/blurts-server/pull/4174/commits/eb173c9828365e8f15dc7a42f030655a77317545).

Reply:
Had to revert and add those back in to have them work in CI.

Reply:
Huh, that's weird; I only have them when redirecting back to Monitor, and the CI succeeds for my PR :thinking: